README.md

```diff
@@ -1,215 +1,10 @@
-"license": "EUPL-1.2",
-```
-
-_Tip: Make sure your newly created package does not contain its own git repository._
-
-Please set the version number of your package dependencies to match the version numbers of
-dependencies of other packages, so that [Lerna](https://github.com/lerna/lerna) can combine these
-dependencies.
-
-Each package must have their own `.eslintignore`, `.gitignore` and `.prettierignore` files. These
-files include things such as the `dist` and `node_modules` folders.
-
-Linting and prettifying is done from the root project, so make sure to remove any configuration
-files (such as `.eslintrc.json` or `.prettierrc.json`) from your project if they are included by
-default.
-
-Testing is done by [Jest](https://jestjs.io/) from the root project. To make this work for your new
-package, remove any testing configuration files, and add a file `jest.config.js` to the root
-directory of your packages with the following content:
-
-```
-const base = require('../../jest.config.base');
-const pack = require('./package.json');
-
-module.exports = {
-  ...base,
-  displayName: pack.name,
-  name: pack.name,
-};
-```
-
-Any additional testing settings - such as `setupFilesAfterEnv` - can be added to this configuration
-file.
-
-### Configuration
-
-Environment variables are loaded from the implementation [@nl-portal/nl-portal-app](./packages/app)
-by default. Possible configuration values are specified in the
-[Config interface](./packages/app/src/interfaces/config.ts).
-
-These values are set to the window object by [config.js](./packages/app/public/config.js), which
-also contains the default values for local development.
-
-When starting the app through Docker, these values can be optionally overridden, i.e.:
-
-```
-docker run --name test -e KEYCLOAK_URL=thekeycloakurl -e KEYCLOAK_REALM=therealrealm -e KEYCLOAK_CLIENT_ID=theclientid -e KEYCLOAK_REDIRECT_URI=theredirecturi GRAPHQL_URI=thegraphqluri -dp 3000:3000 test1
-```
-
-### GraphQL
-
-The implementation [@nl-portal/nl-portal-app](./packages/app) uses
-[Apollo Client](https://www.apollographql.com/docs/react/) through the package
-[@nl-portal/nl-portal-api](./packages/api) to communicate with the GraphQL back-end.
-
-New queries can be added as exported JavaScript variables from separate files
-[in the queries folder](./packages/api/src/queries).
-
-Running `lerna codegen` from the project root will then generate TypeScript code based on these
-query files. For this to succeed, the GraphQL API endpoint specified in
-[codegen.yml](./packages/api/codegen.yml) must be available.
-
-Once the codegen completes, the queries are exported as hooks from
-[@nl-portal/nl-portal-api](./packages/api) and can be imported and used inside a functional
-component:
-
-```
-...
-import {useGetZakenQuery} from '@nl-portal/nl-portal-api';
-
-const CasesPage = () => {
-  const {data, loading, error, refetch} = useGetZakenQuery();
-  ...
-}
+```bash
+npm install --save @nl-portal/nl-portal-localization
```

Comment:
De readme is HEEL veel kleiner geworden. Voor de main readme van een open source project zou ik toch meer informatie willen zien dan alleen dit.

Reply:
Thanks, was idd een foutje.